framework/OpenMod.Core/Plugins/OpenModPluginBase.cs

```diff
@@ -34,7 +38,10 @@ public abstract class OpenModPluginBase : IOpenModPlugin, IAsyncDisposable
         public IRuntime Runtime { get; }
         public IEventBus EventBus { get; }
         protected ILogger Logger { get; set; }
+
+#if NETFRAMEWORK && !NET20
```

Comment:
This is not acceptable, Harmony is supported on all platforms not just .NET Framework

Reply:
i didn't added the other platforms because atm om dont compile to them